content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -46,24 +46,23 @@ Refer to
    gcloud components update
```

Comment:
Add "shell" to the first set of backticks (lines 44, 69, 85, 93, 101, 108, 114, 177, 184, 193, 204, 220)

Reply:
I've been consistently using `bash` though, what's the difference for markdown?

Reply:
I would recommend leaving this as `bash`. My understanding is that our Hugo configuration uses [Chroma](https://github.com/alecthomas/chroma) for syntax highlighting. Chroma does list `bash` as a language that it can highlight, but it does not list `shell`.